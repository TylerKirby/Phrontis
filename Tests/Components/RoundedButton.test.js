import 'react-native';
import React from 'react';
import RoundedButton from '../../App/Components/RoundedButton';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

test('RoundedButton component renders correctly', () => {
  const tree = renderer.create(<RoundedButton text='test' onPress={() => {}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
