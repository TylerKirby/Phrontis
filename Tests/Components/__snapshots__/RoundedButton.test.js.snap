// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundedButton component renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#D81159",
      "borderRadius": 5,
      "height": 45,
      "justifyContent": "center",
      "marginHorizontal": 25,
      "marginVertical": 10,
      "opacity": 1,
      "width": 150,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 14,
        "fontWeight": "bold",
        "marginVertical": 10,
        "textAlign": "center",
      }
    }
  >
    test
  </Text>
</View>
`;
